### ISSUE TITLE
Clarification on feature freeze

### ISSUE THREAD

**Issue by Zenkoh1** [nus-cs2103-AY2425S1/forum#655]
I see that I am allowed to change validation in this case 

>Rejecting valid inputs is a bug and can be fixed

But am I allowed to change the `MESSAGE_CONSTRAINT` to align with my change of validation

**Comment by damithc** [comment]
@Zenkoh1 Yes, but limit the scope of the change to the change in validation.
